includes/reports/pipeline_by_rep.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (params) => {
+  return publish("report_pipeline_by_rep", {
+    description: "Pipeline by Rep",
+    columns: {
+    // DIMENSIONS
+
+
+    // MEASURES
+    
+    },
+    assertions: {
+      //
+    },
+    ...params.defaultConfig
+  }).query(ctx => `
+
+    with opportunitites as (
+        select *
+        from ${ctx.ref("salesforce_opportunities")}
+    
+    ),
+    
+    users as (
+        select *
+        from ${ctx.ref("salesforce_users")}
+    
+    ),
+    
+    pipeline_by_rep as (
```

Comment:
two things:
- this doesn't seem to be by rep as far as I can tell, there's no grouping?
- Based on what the query does, as it has a subset of the inputs to opportunities_extended, shouldn't this just be rolled up into that? Ultimately it seems each row is an oppurtunity in it's current form